cni/network/network.go

```diff
@@ -1049,7 +1147,6 @@ func (plugin *NetPlugin) Delete(args *cniSkel.CmdArgs) error {
 			}
 		}
 
```

Comment:
revert this change

Reply:
this line is on line 1082, just moved to the above